get-started/graph-db-vs-nosql/graph-db-vs-nosql.adoc

```diff
@@ -10,43 +10,43 @@
 {level}
 
 toc::[]
 
+
+== NoSQL Databases
+
 Although unhelpfully named, the NoSQL ("Not only SQL") space brings together many interesting solutions offering different data models and database systems, each suitable for different use cases and shapes of data.
 
 With the advent of the NoSQL movement, the "one-size-fits-all" proposition of large relational systems was replaced by making conscious decisions about finding the right tool for the job.
 
-Most NoSQL systems are *aggregate oriented*, providing only atomic operations within their natural aggregate, which forms only one dedicated view of your data.
-Other projections have to be computed crunching or duplicating your data.
-Focusing on one aggregate at a time allows for easy sharding of bucketed data across a network of machines along the aggregate dimension, for instance the *Document* in Document Databases.
+++++
+<iframe width="680" height="425" src="https://www.youtube.com/embed/5Tl8WcaqZoc" frameborder="0" allowfullscreen></iframe>
+++++
 
-Graph databases handle fine-grained networks of information providing *any perspective* on your data that fits your use-cases.
-Known and trusted from relational systems, transactional guarantees protect updates of that connected data conforming to ACID standards.
+Most NoSQL systems are *aggregate-oriented*, providing only atomic operations within their natural aggregate, which forms only one dedicated view of your data.
+Focusing on one aggregate at a time allows for easy sharding of bucketed data across a network of machines along the aggregate dimension (for instance, the *Document* in document databases), but that means that other projections have to be computed by crunching or duplicating your data.
 
-.NoSQL databases lack relationships
-[quote, "Jim Webber, Ian Robinson", "Graph Databases, O'Reilly"]
-Most NoSQL databases store sets of disconnected aggregates. *This makes it difficult to use them for connected data and graphs.* +
-One well-known strategy for adding relationships to such stores is to embed an aggregate's identifier inside the field belonging to another aggregate — effectively introducing foreign keys. +
-*But this requires joining aggregates at the application level, which quickly becomes prohibitively expensive.*
+****
+[quote, "Graph Databases, O'Reilly"]
+Most NoSQL databases store sets of disconnected aggregates. This makes it difficult to use them for connected data and graphs. One well-known strategy for adding relationships to such stores is to embed an aggregate's identifier inside the field belonging to another aggregate — effectively introducing foreign keys. But, this requires joining aggregates at the application level, which quickly becomes prohibitively expensive.
+****
 
-Let's compare graph databases to other NoSQL systems.
+NoSQL databases lack relationships. Graph databases, on the other hand, handle fine-grained networks of information, providing *any perspective* on your data that fits your use case.
```

Comment:
edit: "**Other** NoSQL databases lack..." to be clear that Neo4j is also a NoSQL database